scipy/stats/_qmc.py

```diff
@@ -914,13 +910,38 @@ class LatinHypercube(QMCEngine):
 
         .. versionadded:: 1.8.0
 
+    orthogonal : bool, optional
+        Orthogonal array based LHS of strength 2 [7]_, [8]_.
+        When used, only ``n=p**2`` points can be sampled, with ``p`` a prime
+        number. It also constrains ``d <= p + 1``.
+        Default is False.
+
+        .. versionadded:: 1.8.0
+
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+
+    LHS is extremely effective on integrands that are nearly additive [2]_.
+    LHS on :math:`n` points never has more variance than plain MC on
+    :math:`n-1` points [3]_. There is a central limit theorem for LHS [4]_,
+    but not necessarily for optimized LHS.
+
+    LHS is an orthogonal array of strength 1.
```

Comment:
I don't have a background in this stuff, so I'm trying to learn what this is from these notes. I'm having a hard time.

>     LHS is an orthogonal array of strength 1.

So an LHS sample is an "orthogonal array" (which is not defined until the next sentence) whether or not `orthogonal=True`? `orthogonal=True` really just ensures that the strength is 2?

Reply:
Yes it is. Any sample is an orthogonal array in the end. MC has strength 0. I see what you mean but I think this might be confusing to use an int instead no? Or it would just take either 1 or 2.

Reply:
I'll leave that to you. Might this be extended to strength 3 or higher? If so, `strength` would be a reasonable name for the parameter. I think it is technically more correct, too, even if the convention is to call a Latin hypercube "orthogonal" only if it has strength 2 or higher. That's a confusing convention, if I'm understanding correctly.

Reply:
`orthogonal` has been changed to `strength`, so this is definitely addressed.